bin/test_travis.sh

```diff
@@ -119,9 +119,15 @@ fi
 if [[ "${TEST_SYMPY}" == "true" ]]; then
     cat << EOF | python
 print('Testing SYMPY, split ${SPLIT}')
-import sympy
-if not sympy.test(split='${SPLIT}'):
+import sympy, coverage
+cov = coverage.Coverage()
+cov.start()
+if not sympy.test(split='${SPLIT}', subprocess=False):
```

Comment:
Also, is there any reason that subprocess=False would make the tests run slower? I'm getting major test slow downs in the first of the split.